cadquery/cq.py

```diff
@@ -1366,6 +1366,34 @@ def sagittaArc(self, endPoint, sag, forConstruction=False):
 
         return self.newObject([arc])
 
+    def radiusArc(self, endPoint, radius, forConstruction=False):
+        """
+        Draw an arc from the current point to endPoint with an arc defined by the sag (sagitta).
+
+        :param endPoint: end point for the arc
+        :type endPoint: 2-tuple, in workplane coordinates
+        :param radius: the radius of the arc
+        :type radius: float, the radius of the arc between start point and end point.
+        :return: a workplane with the current point at the end of the arc
+
+        Given that a closed contour is drawn clockwise;
+        A positive radius means convex arc and negative radius means concave arc.
+        """
+
+        startPoint = self._findFromPoint(False)
+        endPoint = self.plane.toWorldCoords(endPoint)
+        midPoint = endPoint.add(startPoint).multiply(0.5)
+
+        # Calculate the sagitta from the radius
+        length = endPoint.sub(startPoint).Length / 2.0
+        sag = abs(radius) - math.sqrt(radius**2 - length**2)
```

Comment:
As you mentioned, if the `radius < length` then the arc is not possible.
I think the best way to capture this is with a `try` block over the `sag` calculation

```python
try:
    sag = abs(radius) - math.sqrt(radius**2 - length**2)
except ValueError:
    raise ValueError("Arc radius is not large enough to reach the given end-point")
```

Reply:
@jmwright and @fragmuffin: Thanks! Good suggestions, I will try to get this implemented later this weekend.